biogeochem/EDPhysiologyMod.F90

```diff
@@ -2247,43 +2247,57 @@ subroutine fragmentation_scaler( currentPatch, bc_in)
     !
     ! !LOCAL VARIABLES:
     logical  :: use_century_tfunc = .false.
+    logical  :: use_hlm_soil_scalar = .true. ! Use hlm input decomp fraction scalars
     integer  :: j
-    integer  :: ifp                   ! Index of a FATES Patch "ifp"
-    real(r8) :: t_scalar
-    real(r8) :: w_scalar
-    real(r8) :: catanf                ! hyperbolic temperature function from CENTURY
-    real(r8) :: catanf_30             ! hyperbolic temperature function from CENTURY
-    real(r8) :: t1                    ! temperature argument
+    integer  :: ifp                          ! Index of a FATES Patch "ifp"
+    real(r8) :: t_scalar                     ! temperature scalar
+    real(r8) :: w_scalar                     ! moisture scalar
+    real(r8) :: catanf                       ! hyperbolic temperature function from CENTURY
+    real(r8) :: catanf_30                    ! hyperbolic temperature function from CENTURY
+    real(r8) :: t1                           ! temperature argument
     !----------------------------------------------------------------------
 
     catanf(t1) = 11.75_r8 +(29.7_r8 / pi) * atan( pi * 0.031_r8  * ( t1 - 15.4_r8 ))
     catanf_30 = catanf(30._r8)
     
     ifp = currentPatch%patchno 
 
-    if ( .not. use_century_tfunc ) then
-    !calculate rate constant scalar for soil temperature,assuming that the base rate constants 
-    !are assigned for non-moisture limiting conditions at 25C.
-      if (bc_in%t_veg24_pa(ifp)  >=  tfrz) then
-        t_scalar = q10_mr**((bc_in%t_veg24_pa(ifp)-(tfrz+25._r8))/10._r8)
-                 !  Q10**((t_soisno(c,j)-(tfrz+25._r8))/10._r8)
-      else
-        t_scalar = (q10_mr**(-25._r8/10._r8))*(q10_froz**((bc_in%t_veg24_pa(ifp)-tfrz)/10._r8))
-                  !Q10**(-25._r8/10._r8))*(froz_q10**((t_soisno(c,j)-tfrz)/10._r8)
-      endif
+    ! Use the hlm temp and moisture decomp fractions by default
+    if ( use_hlm_soil_scalar ) then
+      
+      ! Calculate the fragmentation_scaler
+      currentPatch%fragmentation_scaler =  min(1.0_r8,max(0.0_r8,bc_in%t_scalar_sisl * bc_in%t_scalar_sisl))
```

Comment:
why is this t_scalar * t_scalar?
Should it be 
```suggestion
      currentPatch%fragmentation_scaler =  min(1.0_r8,max(0.0_r8,bc_in%t_scalar_sisl * bc_in%w_scalar_sisl))
```

Reply:
Yup.  Good catch!  Will fix and rerun my test cases.